lib/libc/common/Kconfig

```diff
@@ -62,7 +61,6 @@ config COMMON_LIBC_CALLOC
 config COMMON_LIBC_REALLOCARRAY
 	bool "Common C library reallocarray"
 	depends on COMMON_LIBC_MALLOC
-	default n if MINIMAL_LIBC && !MINIMAL_LIBC_REALLOCARRAY
```

Comment:
Same here -- leave the line, but remove `&& !MINIMAL_LIBC_REALLOC_ARRAY`

Reply:
updated